projects/packages/search/src/dashboard/class-initial-state.php

```diff
@@ -58,7 +58,8 @@ public function render() {
 	public function get_initial_state() {
 		return array(
 			'siteData'        => array(
-				'WP_API_root'       => $this->get_wp_api_root(),
+				'WP_API_root'       => esc_url_raw( rest_url() ),
+				'wpcomOriginApiUrl' => $this->get_wp_api_root(),
```

Comment:
```suggestion
				'WPCOM_API_ROOT' => $this->get_wp_api_root(),
```
Would creating a parallel between the names `WP_API_ROOT` and `WPCOM_API_ROOT` be valuable here?